tdrs-backend/docker-compose.yml

```diff
@@ -56,7 +56,7 @@ services:
       context: .
       dockerfile: docker/Dockerfile.local
     command: >
-      bash -c "python wait_for_postgres.py &&
+      bash -c "./wait_for_services.sh &&
```

Comment:
The new script also calls the `wait_for_postgres` script so we still wait for the DB to be ready as well as the other services..